objgraph.py

```diff
@@ -72,6 +72,12 @@
     # Python 3.x compatibility
     iteritems = dict.items
 
+try:
+    zip_longest = itertools.izip_longest
+except AttributeError:  # pragma: PY3
+    # Python 3.x compatibility
+    zip_longest = itertools.zip_longest
```

Comment:
can we use six.moves.zip_longest instead?

Reply:
I don't want to introduce non-optional dependencies outside Python's stdlib yet.